### PR TITLE
[GCI] Improve performance by using exists instead of find_by

### PR DESCRIPTION
**Pull request by junron**
#### Describe the changes you have made in this pr -
- Change 3 instances of `find_by` to `exists?` in project policy

[GCI task](https://codein.withgoogle.com/dashboard/task-instances/6132481120534528/)

Please review @tachyons

### CODE REVIEW

**Comment by tachyons** [comment]
Task was to make 3 changes, so that others can also claim the task and fix some

File: app/views/projects/new.html.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div class="container feature">
 <h1>New Project</h1>
 
-<%= render 'form', project: @project %>
+<%= render 'form', project: @project, cached: true %>
```
Comment (tachyons) [inline app/views/projects/new.html.erb]: Caching forms won't make any significant improvement, also it can cause weird bugs

Comment (junron) [reply app/views/projects/new.html.erb]: Ok, I will see what other areas can be cached

**Comment by junron** [comment]
Please have a look @tachyons

**Pull request merged**